Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Compositor/CalibrationData.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System.IO;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.Compositor
+{
+    /// <summary>
+    /// Used for loading/applying camera intrinsics and camera extrinsics obtained through Spectator View's default calibration process.
+    /// </summary>
+    public class CalibrationData : ICalibrationData
+    {
+        private CalculatedCameraIntrinsics cameraIntrinsics;
+        private CalculatedCameraExtrinsics cameraExtrinsics;
+
+        public CalibrationData(string cameraIntrinsicsPath, string cameraExtrinsicsPath)
+        {
+            if (File.Exists(cameraIntrinsicsPath) &&
+                File.Exists(cameraExtrinsicsPath))
+            {
+                cameraIntrinsics = CalibrationDataHelper.LoadCameraIntrinsics(cameraIntrinsicsPath);
+                cameraExtrinsics = CalibrationDataHelper.LoadCameraExtrinsics(cameraExtrinsicsPath);
+
+                if (cameraIntrinsics == null ||
+                    cameraExtrinsics == null)
+                {
+                    Debug.LogError($"Failed to load cameara intrinsics/extrinscs: {cameraIntrinsicsPath}, {cameraExtrinsicsPath}");
+                }
+            }
+            else
+            {
+                Debug.LogError($"Invalid paths provided for camera intrinsics/extrinsics: {cameraIntrinsicsPath}, {cameraExtrinsicsPath}");
+            }
+        }
+
+        public CalibrationData(CalculatedCameraIntrinsics intrinsics, CalculatedCameraExtrinsics extrinsics)
+        {
+            cameraIntrinsics = intrinsics;
+            cameraExtrinsics = extrinsics;
+        }
+
+        /// <inheritdoc />
+        public void SetUnityCameraExtrinstics(Transform cameraTransform)
+        {
+            Matrix4x4 headFromCamera = cameraExtrinsics.ViewFromWorld;
+            cameraTransform.transform.localPosition = cameraExtrinsics.ViewFromWorld.GetColumn(3);
+            cameraTransform.transform.localRotation = Quaternion.LookRotation(cameraExtrinsics.ViewFromWorld.GetColumn(2), cameraExtrinsics.ViewFromWorld.GetColumn(1));
+
+            // Magic offset from Unity's underlying coordinate frame (WorldManager.GetNativeISpatialCoordinateSystemPtr()) and the head pose used for the camera.
+            // Poses are sent in the coordinate frame space because the camera position uses prediction.
```

Comment:
Maybe expand the comment to say "Poses are sent in the coordinate frame space because the Unity camera position uses prediction"